moveit_core/distance_field/include/moveit/distance_field/voxel_grid.h

```diff
@@ -440,12 +437,6 @@ inline double VoxelGrid<T>::getResolution() const
   return resolution_;
 }
 
-template <typename T>
-inline double VoxelGrid<T>::getResolution(Dimension /*dim*/) const
```

Comment:
Why is this template removed? Is it only used by the deprecated functions?

Reply:
The getResolution() function using a Dimension argument was deprecated as per the comment below and this commit https://github.com/ros-planning/moveit2/commit/7448d18dbbcaf20e1b5a44b2b26fe036850bbf82 https://github.com/ros-planning/moveit2/blob/94fea793bbc3609beaaa07f0f3b72056699ac058/moveit_core/distance_field/include/moveit/distance_field/voxel_grid.h#L205

So I also removed the template related to it